robot-base/Dockerfile.base

```diff
@@ -1,8 +1,4 @@
-ARG PYTHON_VERSION
-FROM python:"$PYTHON_VERSION"-alpine
-
-RUN apk add --update --no-cache bash sudo su-exec && \
-  rm -rf /var/cache/* /tmp/* /var/log/* ~/.cache
+FROM quay.io/charter-ctec/python-ubuntu-base:latest 
```

Comment:
try not to use latest (this is used in a couple of places). pin it to a specific version for best practice.

Reply:
👍 Will do.